### ISSUE TITLE
[Bug]: token_counter seems broken

### ISSUE THREAD

**Issue by toniengelhardt** [BerriAI/litellm#1825]
### What happened?

This code returns `3`, when it should be ~11. 

It used to work, but broke with one of the last updates.

![Screenshot 2024-02-05 at 16 05 21](https://github.com/BerriAI/litellm/assets/2703233/8d44c2c4-93cf-4f0f-9dcf-75f03470ad79)

**Comment by toniengelhardt** [comment]
This also breaks the `completion_cost` function.

**Comment by ghost** [comment]
I believe you're just missing the required flag 
`count_response_tokens`

```
def test_token_counter(): 
    from litellm import token_counter

    model = 'gpt-3.5-turbo'  # 'gpt-4-1106-preview'
    text = 'Hello, this is a text with roughly 11 tokens.'

    count = token_counter(model=model, text=text, count_response_tokens=True)

    print(count)
    raise Exception("it worked!")
```

**Comment by ghost** [comment]
also added this scenario in our token_counter logic 

should be fixed in v`1.22.4`
